test/test-applications/integrations/TestApplication.Http/TestApplication.Http.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.0" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.0" />
```

Comment:
Should AdditionalDeps help in case of net6.0/7.0?

Reply:
Good catch. It is not needed. There is no usage of this in the test application.

Removed reference: b9065577861e9fd523ef890a4b8916069e01b4af